pkg/controller/node/nodecontroller.go

```diff
@@ -435,12 +454,58 @@ func (nc *NodeController) doEvictionPass() {
 	}
 }
 
-func (nc *NodeController) doTaintingPass() {
+func (nc *NodeController) doNoScheduleTaintingPass() {
```

Comment:
Yeah - this is not needed, we don't need to rate limit condition-based tainting.